rest-app/src/test/java/com/epam/repair/rest/StreetControllerIntegrationTest.java

```diff
@@ -13,33 +13,37 @@
 import org.springframework.test.web.servlet.request.MockMvcRequestBuilders;
 import org.springframework.test.web.servlet.result.MockMvcResultMatchers;
 
-import static org.hamcrest.Matchers.hasSize;
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+
 import static org.springframework.test.web.servlet.result.MockMvcResultHandlers.print;
 import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.content;
 
 @ExtendWith(SpringExtension.class)
 @SpringBootTest
 @AutoConfigureMockMvc
-public class StreetRestControllerIntegrationTest {
+public class StreetControllerIntegrationTest {
 
     @Autowired
-    private StreetRestController streetRestController;
+    private StreetController streetController;
 
     @Autowired
     private MockMvc mockMvc;
 
     @Test
     public void findAll() throws Exception {
+
+        String expected = loadTestFile("street.json");
+
         mockMvc.perform(MockMvcRequestBuilders.get("/street")
                 .accept(MediaType.APPLICATION_JSON))
                 .andDo(print())
                 .andExpect(MockMvcResultMatchers.status().isOk())
                 .andExpect(content().contentType(MediaType.APPLICATION_JSON))
-                .andExpect(MockMvcResultMatchers.jsonPath("$", hasSize(2)))
-                .andExpect(MockMvcResultMatchers.jsonPath("$").exists())
-                .andExpect(MockMvcResultMatchers.jsonPath("$.[*].streetId").isNotEmpty())
-                .andExpect(MockMvcResultMatchers.jsonPath("$.[*].streetName").isNotEmpty())
-                .andExpect(MockMvcResultMatchers.jsonPath("$.[0].streetId", Matchers.is(1)))
-                .andExpect(MockMvcResultMatchers.jsonPath("$.[0].streetName", Matchers.is("pereulok Brestskikh Diviziy")));
+                .andExpect(content().string(expected));
+    }
+
+    public static String loadTestFile(String street) throws IOException {
+        return FileUtils.readFileToString(FileUtils.toFile(StreetControllerIntegrationTest.class.getClassLoader().getResource(street)), StandardCharsets.UTF_8);
```

Comment:
Can be replaced as test util method

Reply:
Do I need to rename the method or move it outside the class?

Reply:
I think it would be better to move this method outside the class and use it wherever